depmapomics/ccle.py

```diff
@@ -0,0 +1,1423 @@
+    renaming_dict = dict(list(zip(pr_table.CDSID, pr_table.index)))
+    fusions_pr = fusions[
+        fusions[fusionSamplecol].isin(set(renaming_dict.keys()))
+    ].replace({fusionSamplecol: renaming_dict})
+    fusions_filtered_pr = fusions_filtered[
+        fusions_filtered[fusionSamplecol].isin(set(renaming_dict.keys()))
+    ].replace({fusionSamplecol: renaming_dict})
+
+    fusions_pr.to_csv(os.path.join(folder, "fusions_all_profile.csv"), index=False)
+    fusions_filtered_pr.to_csv(
+        os.path.join(folder, "filteredfusions_latest_profile.csv"), index=False
+    )
+
+    if prevdataset == "ccle":
+        prevdataset = tc.get(name=TAIGA_ETERNAL, file="CCLE_fusions_unfiltered")
+
+        print("comparing to previous version")
+        print("new")
+        print(set(fusions[fusionSamplecol]) - set(prevdataset[fusionSamplecol]))
+
+        print("removed")
+        print(set(prevdataset[fusionSamplecol]) - set(fusions[fusionSamplecol]))
+
+    print("changes in fusion names")
+    pf = prevdataset.copy()
+    pf["id"] = pf[fusionSamplecol] + "_" + pf["FusionName"]
+    f = fusions.copy()
+    f["id"] = f[fusionSamplecol] + "_" + f["FusionName"]
+    print(len(set(pf[~pf.id.isin(f.id.tolist())][fusionSamplecol])))
+
+    print("changes in junction read counts")
+    f["sid"] = (
+        f[fusionSamplecol]
+        + "_"
+        + f["FusionName"]
+        + "_"
+        + f["JunctionReadCount"].astype(str)
+    )
+    pf["sid"] = (
+        pf[fusionSamplecol]
+        + "_"
+        + pf["FusionName"]
+        + "_"
+        + pf["JunctionReadCount"].astype(str)
+    )
+    print(len(set(pf[~pf.sid.isin(f.sid.tolist())][fusionSamplecol])))
+
+    # taiga
+    print("uploading to taiga")
+    tc.update_dataset(
+        dataset_permaname=taiga_dataset,
+        changes_description="new " + sampleset + " release!",
+        upload_files=[
+            {
+                "path": "output/" + sampleset + "/filteredfusions_latest.csv",
+                "name": "filtered_fusions",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": "output/" + sampleset + "/fusions_all.csv",
+                "name": "fusions-all",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": "output/" + sampleset + "/filteredfusions_latest_profile.csv",
+                "name": "filtered_fusion-profile",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": "output/" + sampleset + "/fusions_all_profile.csv",
+                "name": "fusion-profile",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+        ],
+        dataset_description=dataset_description,
+    )
+    print("done")
+    return fusions
+
+
+def cnPostProcessing(
+    trackerobj,
+    gumbo,
```

Comment:
there is two required here?

I think none should be required in the CCLE.py file

Reply:
there should be one by default

Reply:
and what is the diff between the two?